plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/impl/api/flatten.kt

```diff
@@ -14,7 +14,7 @@ import org.jetbrains.kotlinx.dataframe.plugin.impl.toPluginDataFrameSchema
 class FlattenDefault : AbstractSchemaModificationInterpreter() {
     val Arguments.receiver by dataFrame()
     val Arguments.keepParentNameForColumns: Boolean by arg(defaultValue = Present(false))
-    val Arguments.separator: String by arg(defaultValue = Present("."))
+    val Arguments.separator: String by arg(defaultValue = Present("_"))
```

Comment:
is this reflects our new convention about separator in column names?

Reply:
Yes :)